runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -54,19 +54,20 @@ TSHighlighter.hl_map = {
     ["include"] = "Include",
 }
 
-function TSHighlighter.new(query, bufnr, ft)
+function TSHighlighter.new(query, bufnr, ft, parser)
```

Comment:
I suppose we can change the signature to be `TSHighlighter.new(parser, query)`. This is not likely to be used by end-users directly anyway.

Reply:
I agree, just need to notify the few people using we broke it (again...).